reservation_units/tasks.py

```diff
@@ -61,5 +61,5 @@ def refresh_reservation_unit_product_mapping(reservation_unit_pk) -> None:
     # Remove product mapping if merchant is removed
     if reservation_unit.payment_product is not None and payment_merchant is None:
```

Comment:
I would suggest to change this `if` condition to this to:
`if reservation_unit.payment_product and not payment_merchant:`

obviously in every case `is not None` does not equal plain `if <something>` but in this case it will (and for the most cases).

Reply:
I know its not part the changeset but while you at it :innocent:

Reply:
Sure, I'll change this! ☺️ 

I'm still a bit confused when in Python it is OK to check truthy and when it is recommended to explicitly check None value 🤔 I've seen some mixed guidelines 🤪